meta-mel/recipes-connectivity/connman/connman_%.bbappend

```diff
@@ -1,2 +1,8 @@
 VIRTUAL-RUNTIME_bluetooth-stack ?= "bluez4"
 PACKAGECONFIG[bluetooth] := "${@d.getVarFlag('PACKAGECONFIG', 'bluetooth').replace('bluez4', '${VIRTUAL-RUNTIME_bluetooth-stack}')}"
+
+# Use PACKAGECONFIG handling for rdepends
+RDEPENDS_${PN} = "dbus xuser-account"
+PACKAGECONFIG[wifi] = "--enable-wifi, --disable-wifi,,wpa-supplicant"
+PACKAGECONFIG[bluetooth] = "--enable-bluetooth, --disable-bluetooth,,bluez5"
```

Comment:
You do realize that line 2 showed exactly what you should have here, right? Look up a few lines, you'll see it replacing bluez4 with `${VIRTUAL-RUNTIME_bluetooth-stack}`. This is exactly what you should use here rather than bluez5, as this is a runtime dependency, not build time (virtual/libbluetooth is build time).